portfolio/src/main/webapp/script.js

```diff
@@ -16,17 +16,31 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['I have lived in the same house my whole life!', 
-      'I hate tomatoes!', 'I do aerial silks!', 
-      'I studied abroad twice!', 'I took the picture above in New Zealand!', 
-      'I went bunjee jumping and skydiving on the same day!',
-      'The picture to the right is from Barcelona!'];
+  const greetings = [
+    "I have lived in the same house my whole life!",
```

Comment:
Seems like prettier uses double quotes by default, but you can override this with https://prettier.io/docs/en/options.html#quotes

to match [the style guide](https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes).